### ISSUE TITLE
Show device warnings/errors

### ISSUE THREAD

**Issue by ebakir** [OpenEMS/openems#30]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
```

### Desired functionality.

Warnings and errors from connected devices (e.g. "warnings"-Channel) should be displayed on the main screen. Also exceptions from OpenEMS should be forwarded to the UI and shown.

**Comment by ebakir** [comment]
First:
capture exceptions centralized

**Comment by sfeilmeier** [comment]
Closed in #1162
